applications/API/forms.py

```
@@ -0,0 +1,495 @@
+            "reporter_email": "Your email address (required if you want to be contacted)",
+            "rationale_for_review": "Provide a reason for your review",
+        }
+
+        help_texts = {
+            "insult_reference_id": "Select the insult you want reviewed review",
+            "reporter_first_name": "Your first name (required unless anonymous)",
+            "reporter_last_name": "Your last name or initial (required unless anonymous)",
+            "post_review_contact_desired": "Check if you want to be contacted with the review results",
+            "reporter_email": "Your email address (required if you want to be contacted)",
+            "rationale_for_review": "Provide a basis for review of the insult (minimum 70 characters)",
+        }
```

Ruff (0.12.2)

Class field `help_texts` is defined multiple times

Remove duplicate field definition for `help_texts`

(PIE794)